tools/assets/stylesheets/avaiga.css

```diff
@@ -1,25 +1,40 @@
+@import 'partials/_grid.css';
+
 :root {
   --md-primary-fg-color: #293183;
   --md-primary-fg-color--light: #f6921b;
   --md-primary-fg-color--dark: #ec693c;
+  --md-paper-bg-color: #fff;
+  --transition-duration-base: 400ms;
+  --tooltip-show-delay: 400ms;
+}
+[data-md-color-scheme="slate"] {
+  --md-hue: 240;
+  --md-paper-bg-color: #140a1e;
 }
+
 .md-header__button.md-logo img {
   width: 48px;
   height: 48px;
 }
 
-[data-md-color-scheme="slate"] {
-  --md-hue: 240;
+.md-content{
```

Comment:
Please make the CSS code homogeneous: space before curly brace always or never.

Reply:
Done.